.github/workflows/run-snapshot-generation.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: snapshot-recording-failure-report
-          path: /home/runner/work/Loudius/Loudius/components/build/reports/tests/testDebugUnitTest/
+          path: components/build/reports/tests/testDebugUnitTest/
```

Comment:
info: using relative path, instead of path on runner to ensure if a directory changed because of CI our tests won't fail